enterprise/internal/batches/store/worker_workspace_execution_test.go

```diff
@@ -114,17 +106,62 @@ stdout: {"operation":"CACHE_AFTER_STEP_RESULT","timestamp":"2021-11-04T12:43:19.
 		}
 	}
 
+	assertWorkspaceChangesets := func(t *testing.T, job *btypes.BatchSpecWorkspaceExecutionJob, want []int64) {
+		t.Helper()
+		w, err := s.GetBatchSpecWorkspace(ctx, GetBatchSpecWorkspaceOpts{ID: job.BatchSpecWorkspaceID})
+		if err != nil {
```

Comment:
Wanna spice things up and try out [`testify/assert`](https://pkg.go.dev/github.com/stretchr/testify/assert?utm_source=godoc) for these assertions? 😀 

```suggestion
		assert.NoError(t, err)
```